lib/posts.ts

```diff
@@ -69,3 +77,28 @@ export const getPostData = async (id: string) => {
     ...(matterResult.data as MatterResultData),
   }
 }
+
+export const getPostDataForFeed = async () => {
+  const allPostsPromise = getSortedPostData().map(async (post) => {
+    const { id, title, description, date } = post
+    const { contentHtml } = await getPostData(id)
+
+    return {
+      id,
+      contentHtml,
+      description,
+      date,
+      title,
+    }
+  })
+
+  const allPosts: PostForFeed[] = await Promise.all(allPostsPromise)
+
+  return allPosts.sort((post1, post2) => {
+    if (post1.date < post2.date) {
+      return 1
+    } else {
+      return -1
+    }
+  })
```

Comment:
82行目で getSortedPostData でソート済みのポスト一覧を取っているのでここでまた新たにソートしなくても大丈夫かなと思いました！
動作確認したところ allPosts とソートした allPosts で特に順番も変わりませんでした。

```suggestion
  return allPosts
```

Reply:
@nabeliwo 完全に考慮から抜けていました、ありがとうございます:pray: